.github/ISSUE_TEMPLATE/issue--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 'Issue: Feature request'
+about: Suggest an idea/extension for Hazel
+title: Add a relevant feature title here
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+ **Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+ **Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+ **Additional context**
+Add any other context or screenshots about the feature request here. Any relevant information for platform specific features could be usefull.
```

Comment:
usefull -> useful

Reply:
Fixed typo in https://github.com/TheCherno/Hazel/pull/80/commits/bda4f8f9e22f8887fb4b046c185349f9863368a4.